packages/arb-token-bridge-ui/src/hooks/useIsTestnetMode.ts

```diff
@@ -3,9 +3,10 @@ import { useLocalStorage } from 'react-use'
 const testnetModeLocalStorageKey = 'arbitrum:bridge:settings:testnetMode'
 
 export const useIsTestnetMode = () => {
-  const [isTestnetMode = false, setIsTestnetMode] = useLocalStorage<boolean>(
-    testnetModeLocalStorageKey
+  const [isTestnetMode, setIsTestnetMode] = useLocalStorage<boolean>(
+    testnetModeLocalStorageKey,
+    false
   )
 
-  return [isTestnetMode, setIsTestnetMode]
+  return [isTestnetMode, setIsTestnetMode] as const
```

Comment:
why is this a const?

Reply:
typescript doesn't interpret types correctly otherwise, this was suggested by @spsjvc